src/ModalContent/ModalContent.stories.js

```diff
@@ -0,0 +1,89 @@
+import AriaModal from 'react-aria-modal';
```

Comment:
This import statement should also be shown in the code markup that is outputted in https://deploy-preview-64--decanter-react.netlify.app/?path=/docs/composite-modal-content--in-modal as it is not clear what the `AriaModal` component is in just the UI.